flights/berks_ceh_hh_epi_services_flight.yaml

```diff
@@ -0,0 +1,380 @@
+# change limit for prod: select e."PersonalID", e."EnrollmentID", e."EntryDate", e."HouseholdID", e."DisabledHoH", e."RelationshipToHoH", s."ServicesID", s."DateProvided", s."RecordType", s."TypeProvided" from enrollment as e left join services as s on e."EnrollmentID"=s."EnrollmentID" AND e."PersonalID"= s."PersonalID" limit 5000; 
+
+#intellij: "select e.\"PersonalID\", e.\"EnrollmentID\", e.\"EntryDate\", e.\"HouseholdID\", e.\"DisabledHoH\", e.\"RelationshipToHoH\",  s.\"ServicesID\", s.\"DateProvided\", s.\"RecordType\", s.\"TypeProvided\" from enrollment as e left join services as s on e.\"EnrollmentID\"=s.\"EnrollmentID\" AND e.\"PersonalID\"= s.\"PersonalID\" limit 5000;"
+
+entityDefinitions: 
+  client: 
+    fqn: "general.person"
+    entitySetName: "BerksCEHClients"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "PersonalID"
+    ol.datasource:
+        type: "ol.datasource"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Berks County CEH"
+    conditions:
+    - !<conditions.BooleanIsNullCondition>
+      column: "PersonalID"
+      reverse: false
+    name: "client"
+
+  household: 
+    fqn: "demographics.household"
+    entitySetName: "BerksCEHHouseholds"
+    propertyDefinitions:
+      household.id:
+        type: "household.id"
+        column: "HouseholdID" # enrollment table
+      ol.disabledheadofhousehold: 
+        type: "ol.disabledheadofhousehold"
+        column: "DisabledHoH" #enrollment
+        transforms: 
+        # - !<transforms.ParseBoolTransform> {}
+        - !<transforms.ReplaceTransform>
+          target: ["0", "1"]
+          goal: [false, true]
+    name: "household"
+
+  enrollment: 
+    fqn: "ol.episode"  
+    entitySetName: "BerksCEHEnrollments"
+    propertyDefinitions:
+      ol.id: 
+        type: "ol.id"
+        column: "EnrollmentID"
+      ol.effectivedate: 
+        type: "ol.effectivedate"
+        column: "EntryDate"
+        transforms: 
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+    name: "enrollment"
+
+  # organization:
```

Comment:
Why are organization and program commented out? They seem like good things to put in - along with their associations.